Content.Server/Chat/ChatCommands.cs

```diff
@@ -72,4 +78,81 @@ public void Execute(IConsoleShell shell, IPlayerSession player, string[] args)
             chat.SendOOC(player, string.Join(" ", args));
         }
     }
+
+    internal class SuicideCommand : IClientCommand
+    {
+        public string Command => "suicide";
+
+        public string Description => "Commits suicide";
+
+        public string Help => "suicide";
```

Comment:
I think it would be good to add details about how the suicide item is chosen here. You can do that on a new line in the string, like `"suicide\n<your stuff here>"`.